src/examples/dialog/dialog-form.tsx

```diff
@@ -25,6 +25,16 @@ export class DialogFormExample {
     @State()
     private isConfirmationOpen = false;
 
+    @State()
+    private formatNumber = true;
```

Comment:
This does not need to be a `State()` variable, or even a variable at all since we never change it